.circleci/config.yml

```diff
@@ -25,6 +25,14 @@ jobs:
           fce build --features fce
 
           cd ..
```

Comment:
to avoid `cd`-hell, you can use sub-shells:
```
(
  cd stepper
  fce build --features fce
)

(
  cd stepper-lib/tests/security_tetraplets/auth_module
  fce build
)

(
  cd stepper-lib/tests/security_tetraplets/log_storage
  fce build
)

cargo fmt ...
```

and so on. This way, CWD is preserved only inside parentheses